communication.rst: add a note about Matrix (#670)
ansible/ansible-documentation: docs/docsite/rst/community/communication.rst

Real-time chat
==============

For real-time interactions, conversations in the Ansible community happen over two chat protocols: Matrix (recommended) and IRC.

> Major Matrix and IRC channels exchange messages, so, to use them, you can choose whichever protocol you prefer.

Reviewer: ```suggestion
The main Matrix and IRC channels exchange messages.
This means you can choose whichever protocol you prefer for the main channels.
```

Author: @oraNod they are not bridged now as Greg said, that's why i don't call it `bridge`:)
Could you  please adjust your comment taking ^ into consideration? I like other parts UPDATE: i mean "Many of the main .."

Reviewer 2: 5 out of 35 rooms is not "Many" either :)

Author: Let's maybe leave it as is then:)

Reviewer: Made a change. Not sure "major" is quite right tbh. Also it's not really clear what "exchange messages" is supposed to mean.

Messages in the main channels are available on both Matrix and IRC, which allows you to use either protocol and participate with all users.
However in most of the other channels, messages are available on either Matrix or IRC but not both.

Author: @oraNod committed, thanks, PTAL

Real-time chat
==============

For real-time interactions, conversations in the Ansible community happen over two chat protocols: Matrix (recommended) and IRC.
The main Matrix and IRC channels exchange messages.
This means you can choose whichever protocol you prefer for the main channels.

.. note::

>   Although you can choose any of the protocols, please take into account that many Ansible communities use only Matrix.

Reviewer: ```suggestion
  Although you can choose either Matrix or IRC, please take into account that many Ansible communities use only Matrix.
```

Author: looks much better, thanks, committed, PTAL
